policies/cla.html

```diff
@@ -30,10 +30,17 @@
 	  the terms under which intellectual property has been contributed
 	  to OpenSSL and thereby allow us to defend the project should
 	  there be a legal dispute regarding the software at some future
-	  time.</p>
+	  time.
+          </p>
+
+          <p>
+          A submission is considered trivial if the author and all reviewers
```

Comment:
That's not correct.  Everyone has to agree, and it also has to be trivial under copyright law.

Reply:
I guess the [source for this text](https://github.com/openssl/web/blob/master/policies/committers.html#L124) is wrong too :)

> All authors, including committers, must have current <a href="/policies/cla.html">CLAs</a> on file. A CLA is not required for trivial contributions (e.g. the fix of a spelling mistake). If all reviewers as well as the original author agree that the submission is trivial, the commit text should include "CLA: trivial."

I.e. [`policies/committers.html`](https://github.com/openssl/web/blob/master/policies/committers.html) will require a similar update.

Reply:
No that text is correct because it points out an *example* of a trivial contribution, it is not claiming to be a *definition* of trivial.

Reply:
That's not how I read the text.  There is no definition of _trivial_ anywhere else.

Reply:
Well, you're reading it wrong.  The "e.g." means *for example.*  You are right that there is no definition of trivial, just an example of something that is.  That was deliberate.